safe-global/safe-wallet-web
# fix: Refetch submission data when submitting submission

## What it solves

- Hides the survey banner if a user already submitted it

## How this PR fixes it

- The RTK Query was caching the submission result which contained `completionDate: null`. After reconnecting with the wallet it still used that value and determined to show the banner to the user again. This now calls `refetch` when the user submits the banner which should cause the RTK query to update its value and hide the banner next time a wallet is connected.

## How to test it

1. Open an elligible Safe and connect with one of the owners
2. Observe the banner showing up
3. Press the submit button in the banner
4. Go back to the app
5. Disconnect your wallet and reconnect again
6. Observe the banner not showing up again

## Checklist
